From d7a34bb388e33d16260b67275cdb58f9c877d324 Mon Sep 17 00:00:00 2001
From: Changqing Li <changqing.li@windriver.com>
Date: Tue, 27 Nov 2018 15:27:47 +0800
Subject: [PATCH] From 6d24365f0828185fd1bb4d199209ca07eb95c41d Mon Sep 17
 00:00:00 2001 From: Khem Raj <raj.khem@gmail.com> Date: Fri, 24 Aug 2018
 06:24:36 +0000 Subject: [PATCH] Do not hardcode path for pkg.m4

Upstream-Status: Pending

Signed-off-by: Khem Raj <raj.khem@gmail.com>

update patch to version 5.1.6
Signed-off-by: Changqing Li <changqing.li@windriver.com>

---
 configure.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.in b/configure.in
index 07c2051..e85d718 100644
--- a/configure.in
+++ b/configure.in
@@ -12,7 +12,7 @@ define([AC_CACHE_SAVE], )dnl
 AC_INIT(.autofs-5.1.8)
 
 # for pkg-config macros
-m4_include([/usr/share/aclocal/pkg.m4])
+m4_include([pkg.m4])
 
 #
 # autofs installs by default in /usr
